packages/@o3r/eslint-config-otter/eslint.local.config.mjs

```diff
@@ -23,5 +23,29 @@ export default [
         ]
       }
     }
+  },
+  {
+    name: '@o3r/eslint-config-otter/overrides',
+    rules: {
+      '@o3r/json-dependency-versions-harmonize': [
+        'error',
+        {
+          ignoredDependencies: [
+            'eslint',
+            '@typescript-eslint/eslint-plugin',
```

Comment:
Ok, then maybe you can add a `todo` to remove this config on the next major

Reply:
Don't we want to deprecate the whole package because some stuff will not work with eslintrc later maybe
I would keep these versions until we delete this package no ?

Reply:
As discussed this is depending of if the new package is usable by the project that did not migrate to flat config.
If it is not I think we should keep updating this package until removal.